### PR TITLE
Remove Informational vs Standard track distinction

### PR DESCRIPTION
**Pull request by msfeldstein**
Remove the Information vs Standard track distinction. It seems confusing and I'm not sure what the value prop of having the complexity is.

### CODE REVIEW

File: ecosystem/README.md [hunk 1]
```diff
@@ -1,14 +1,5 @@
...
 
-## SEP Tracks
-* **Informational** — A SEP on the `Informational` track is one that is open to adoption by the
-  ecosystem, but has not been formally standardized by SDF, and is not endorsed by SDF for
-  adoption. Typically a SEP can start as `Informational` to gain traction within the ecosystem
-  before moving to the `Standards` track.
-* **Standard** — A SEP on the `Standards` track is one that aims for formal standardization and
-  endorsement by SDF for adoption. Typically a SEP Standard has a higher bar towards acceptance,
-  and it requires approval by 2 SDF members of the SEP Team.
-
```
Comment (leighmcculloch) [inline ecosystem/README.md]: The word `Standard` is mentioned in the SEP Status Terms section. I can't comment on the lines directly, but its line 12, line 15, line 19.

Comment (msfeldstein) [reply ecosystem/README.md]: updated

**Comment by MonsieurNicolas** [comment]
As this distinction was made to reduce the friction for non SDF proposals, I'd ask @rice2000 what he thinks about it

**Comment by tomerweller** [comment]
I agree that this might be confusing but would not remove it without addressing the issue it tries to solve. There needs to be a way for people in the ecosystem to inform the community of a convention they are using, without going through the hassle of the standardization process. This is a similar model to [EIPs](https://eips.ethereum.org/).

File: ecosystem/README.md [hunk 2]
```diff
@@ -18,49 +9,47 @@
...
+| [SEP-0008](sep-0008.md) | Regulated Assets | Interstellar | Final |
+| [SEP-0009](sep-0009.md) | Standard KYC / AML Fields | SDF | Active |
+| [SEP-0010](sep-0010.md) | Stellar Web Authentication | Sergey Nebolsin, Tom Quisel | Active |
+| [SEP-0011](sep-0011.md) | Txrep: Human-Readable Low-Level Representation of Stellar Transactions | David Mazières | Active |
+| [SEP-0012](sep-0012.md) | Anchor/Client Customer Info Transfer | Interstellar | Active |
+| [SEP-0020](sep-0020.md) | Self-verification of validator nodes | Johan Stén | Active |
 
 ### Draft Proposals
 
 | Number | Title | Author | Track | Status |
```
Comment (rice2000) [inline ecosystem/README.md]: Delete | Track | column heading so table renders properly

**Comment by rice2000** [comment]
Currently, the claim is that: "The bar for accepting an `Informational` SEP is much lower than one for a `Standard`..."  However, the only real difference I see is that the `Standard` track requires 3 approvals, whereas an `Informational` requires 2.  That doesn't seem like a meaningful difference to me, and it doesn't really seem to reduce friction.

Reducing complexity _does_ reduce friction, so I'm all for getting rid of the tracks.

**Comment by msfeldstein** [comment]
Yeah i get the point of the informational track, and in this case i feel it would be better to have something just totally separated.  If it's purely informational we don't need a real review or to go through the stages, we would just want to ensure the proposals are written well and are at least somewhat useful to have in this repo.

Another solution might be to have informational seps be another status vs a separate track. These could go `Draft -> Awaiting Decision -> Informational` instead of going to active or final.

**Comment by tomquisel** [comment]
Is there an example of an informational SEP? Or one being concocted somewhere?

**Comment by msfeldstein** [comment]
There are a few draft proposals here: https://github.com/stellar/stellar-protocol/tree/master/ecosystem, and one that looked like sep6 that we ended up rejecting since sep6 subsumed it.

**Comment by msfeldstein** [comment]
https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0022.md
and 
https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0021.md

**Comment by leighmcculloch** [comment]
This PR looks like it has been superseded by #555. I'm going to close it because of that. If I'm making a mistake by closing this, sorry!, please reopen.

**Pull request closed without merging**